### ISSUE TITLE
TypeError on conversion

### ISSUE THREAD

**Issue by sundarv85** [wolph/alfred-converter#55]
![image](https://github.com/wolph/alfred-converter/assets/11090193/ae4147f2-d609-4c5e-91b4-3af26e12d017)

I am getting this error. I just installed the latest version in M1. Any suggestions?

**Comment by wolph** [comment]
That error occurs with older (3.8 and older) versions of Python. You need to have Python 3.9 or newer installed on your system for the extension to work.

I would suggest installing a newer Python version through homebrew: https://docs.python-guide.org/starting/install3/osx/

**Comment by sundarv85** [comment]
I do have python 3.11, but it is with python3

```
$ python3 --version
Python 3.11.5
```

```
python -v
pyenv: python: command not found

The `python' command exists in these Python versions:
  3.11.3

Note: See 'pyenv help global' for tips on allowing both
      python2 and python3 to be found.
```

python command is not found.

The python 3.11 is installed through homebrew only.

**Comment by wolph** [comment]
I've just checked (my mac was out of commission for a little while) and the current script expects python to be in `/usr/bin/python3`. That is the system version that's bundled with OS X. On my current system (I'm running 12.7) it's at 3.9 but from this I'm gathering that you are running an older version of OS X?

In any case, I've uploaded a new release with compatibility for Python 3.7 and higher so it should work without any issues on your system :)
